install/disconnected_install.adoc

```diff
@@ -195,24 +195,50 @@ endif::[]
 +
 [source, bash]
 ----
+$ docker pull registry.access.redhat.com/openshift3/apb-base:<tag>
+$ docker pull registry.access.redhat.com/openshift3/apb-tools:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-attacher:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-driver-registrar:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-livenessprobe:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/image-inspector:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-maven-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/local-storage-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/logging-eventrouter:<tag>
+$ docker pull registry.access.redhat.com/openshift3/manila-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/mariadb-apb:<tag>
+$ docker pull registry.access.redhat.com/openshift3/metrics-schema-installer:<tag>
```

Comment:
Two `metrics-schema-installer` images present in this section. And this one should be for section "additional centralized log aggregation and metrics aggregation components".